### PR TITLE
[1LP][RFR] Refarctor Nuage fixture

### PR DESCRIPTION
**Pull request by aljazkosir**
Refactor Nuage fixture so we have an option to reuse it with different test configuration.

/cc @miha-plesko

### CODE REVIEW

File: cfme/fixtures/nuage.py [hunk 1]
```diff
@@ -45,9 +44,18 @@ def with_nuage_sandbox(networks_provider):
     box['l2_group'] = box['l2_domain'].create_child(
         nuage.vspk.NUPolicyGroup(name=random_name()))[0]
 
+    return box
+
+
+@pytest.fixture
+def with_nuage_sandbox(networks_provider):
+    nuage = networks_provider.mgmt
+    sandbox = create_nuage_sandbox(networks_provider)
```
Comment (miha-plesko) [inline cfme/fixtures/nuage.py]: Please pass `nuage` handle directly to the `create_nuage_sandbox` function instead passing `networks_provider` fixture. We want the auxilary function be as pure as possible.

File: cfme/fixtures/nuage.py [hunk 2]
```diff
@@ -5,10 +5,9 @@
 from cfme.utils.log import logger
 
 
-@pytest.fixture
-def with_nuage_sandbox(networks_provider):
+def create_nuage_sandbox(networks_provider):
```
Comment (miha-plesko) [inline cfme/fixtures/nuage.py]: May I suggest to name it `create_basic_sandbox` instead? In the future we plan to come up with an extended sandbox as well (with Nuage templates and all) where we'll want to first create this basic setup and then extend it.

Comment (miha-plesko) [reply cfme/fixtures/nuage.py]: Also, please modify the method to accept `nuage` as parameter, as suggested by the comment below.

Comment (aljazkosir) [reply cfme/fixtures/nuage.py]: I modified both function name and input parameter as suggested.

**Pull request merged**